<?xml version="1.0"?>
<launch>

    <node name="cloud_save" pkg="pointcloud_usage" type="cloud_save"  output="screen" respawn="false" respawn_delay="0">
        <param name="topic_name" value="/extract_1"/>
        <!--param name="topic_name" value="/cloud_without_segmented"/-->
<<<<<<< HEAD
        <param name="save_name" value="/home/tsuchida/yes_1.pcd"/>
=======
        <param name="save_name" value="/home/ericlab/ah.pcd"/>
>>>>>>> 43a7e681882d064e4054e72304136882d89e48ea
        <param name="pub_topic_name" value="/shonya_3"/>
    </node>

</launch>